src/ranges.js

```diff
@@ -396,3 +396,44 @@ export const timeRangesToArray = (timeRanges) => {
 
   return timeRangesList;
 };
+
+/**
+ * Determines if two time range objects are different.
+ *
+ * @param {TimeRange} a
+ *        the first buffered object to check
+ *
+ * @param {TimeRange} b
+ *        the second buffered object to check
+ *
+ * @return {Boolean}
+ *         Whether the time range objects differ
+ */
+
+export const isRangeDifferent = function(a, b) {
```

Comment:
I originally had this as a function in `playback-watcher` but it seems generic enough to pull into `ranges`.